src/main/java/org/fineract/messagegateway/sms/api/SmsApiResource.java

```diff
@@ -42,9 +52,22 @@
 public class SmsApiResource {
 
 	//This class sends TRANSACTIONAL & PROMOTIONAL SMS
+
 	private SMSMessageService smsMessageService ;
+
 	private static final Logger logger = LoggerFactory.getLogger(SmsApiResource.class);
 
+	@Autowired
+	private TelerivetMessageProvider telerivetMessageProvider;
+
+	private SmsOutboundMessageRepository smsOutboundMessageRepository;
```

Comment:
How is the object created? Should it be auto wired?

Reply:
this is a redundant  object declaration. I missed to remove it as it is not serving any purpose now

Reply:
yeah just updating this PR itself

Reply:
Yes. Thank you.